src/test/java/com/mentoring/framework/Config.java

```diff
@@ -30,7 +30,7 @@
         PORT = prop.getProperty("port");
         if (System.getenv(ENVIRONMENT).equalsIgnoreCase(SKY_XPLORE_REMOTE)) {
             //TODO: Replace static IP address with dynamic get from emails
-            BASE_URL = ("http://").concat("62.165.192.145");
+            BASE_URL = "http://".concat("62.165.192.145");
```

Comment:
Why do you use the IP address hard coded? We should read it from a resource file.

Reply:
Please read: TODO above
It should be handled in TAF-10: https://trello.com/c/aEcPyysn/10-get-the-latest-ip-address-of-the-application-from-the-automatically-sent-email-via-the-emailutilsjava